docs/references/http-websocket-apis/public-api-methods/server-info-methods/feature.md

```diff
@@ -0,0 +1,142 @@
+---
+html: feature.html
+parent: server-info-methods.html
+seo:
+    description: Get information about protocol amendments.
+labels:
+  - Blockchain
+  - Core Server
+---
+
+# feature
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/xrpld/rpc/handlers/Feature1.cpp "Source")<br/>
+
+The `feature` command returns information about [amendments](../../../../concepts/networks-and-servers/amendments.md) this server knows about, including whether they are enabled and if the server knows how to apply the amendments.
+
+This is the non-admin version of the [`feature` admin command](../../admin-api-methods/status-and-debugging-methods/feature.md). It follows the same formatting as the _admin_ command, but hides potentially sensitive data.
+
+{% badge href="https://github.com/XRPLF/rippled/releases/tag/2.2.0" %}New in: rippled 2.2.0{% /badge %}
+
+## Request Format
+
+An example of the request format:
+
+{% tabs %}
+
+{% tab label="WebSocket" %}
+```json
+{
+  "command": "feature",
+  "feature": "4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373"
+}
+```
+{% /tab %}
+
+{% tab label="JSON-RPC" %}
+```json
+{
+    "method": "feature",
+    "params": [
+        {
+            "feature": "4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373"
+        }
+    ]
+}
+```
+{% /tab %}
+
+{% tab label="Commandline" %}
+```sh
+#Syntax: feature [<feature_id>]
+rippled feature 4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373
+```
+{% /tab %}
+
+{% /tabs %}
+
+{% try-it method="feature" /%}
```

Comment:
The Try-It button in the preview doesn't reflect the new method.

Reply:
Ah, looks like I missed adding `feature` to the `Try-It` configuration file. This is fixed now, see commit [dccc5aa65c76e13cdad180a032b5f3df9d054156](https://github.com/XRPLF/xrpl-dev-portal/pull/3040/commits/dccc5aa65c76e13cdad180a032b5f3df9d054156)